tensorflow/compiler/xla/service/plaidml/tests/plaidml_conv_io_generator.py

```diff
@@ -0,0 +1,81 @@
+import tensorflow as tf
+import numpy as np
+import os
+import glob
+import itertools
+import shutil
+
+np.set_printoptions(threshold = 125**2)
+tf.compat.v1.disable_eager_execution()
+
+i_sizes = [[1, 224, 224, 3]]
+k_sizes = [[7, 7, 3, 64]]
+strides = [(1, 1)]
+paddings = [[[0, 0], [3, 3], [3, 3], [0, 0]]]
+
+# [[0, 0], [0, 0], [0, 0], [0, 0]],
+#             [[0, 0], [1, 1], [1, 1], [0, 0]],
+#             "VALID",
+#             "SAME",
+            
+dilations = [(1, 1)]
```

Comment:
In the future, we could definitely expand this out to test more combinations of strides, paddings, and dilations.

Reply:
Yep that's the plan (and conv3d) but it needed an input that big to generate the segfault, and if I included other tests in the same header it would be a pretty massive file so I left them out for now

Reply:
Right -- I have Jeremy's changes pulled in on a separate branch, mind if I merge those changes in here so we can more accurately assess the behavior? I was under the impression that single convolutions do not run into errors anymore.

Reply:
Yeah go for it! Yeah I think once those changes are added it should pass.